CHANGELOG.md

```diff
@@ -42,6 +42,7 @@
 - `fix` Fixed problems with url rules #351 (dmeroff)
 - `enh` Regenerate `auth_key` after blocking the user (dmeroff)
 - `enh` Improved registration process #236 (dmeroff)
+- `enh` Allow fast login without password when in DEBUG mode (thyseus)
```

Comment:
I'm sure this is wrong place for this change log :smile: It's about version 0.9.5, however latest version going to be released is 0.9.10.